client/mm/libxc/coinbase.go

```diff
@@ -205,11 +206,8 @@ func (c *cbWSConn) handleSubscriptionMessage(b []byte) {
 	var subbed bool
 	for channel, productIDs := range msg.Events[0].Subscriptions {
 		if channel == c.channel {
-			for _, productID := range productIDs {
-				if subscribed(productID) {
-					subbed = true
-					break
-				}
+			if slices.ContainsFunc(productIDs, subscribed) {
+				subbed = true
 			}
```

Comment:
I think maybe we wanted to break out of all of this if we found subbed, so I think we should keep break in there, and this was sort of wrong before:
```suggestion
			if slices.ContainsFunc(productIDs, subscribed) {
				subbed = true
				break
			}
```

Reply:
Thanks for your suggestions. Modified.